opamp/observiq/observiq_client.go

```diff
@@ -85,8 +85,14 @@ func NewClient(args *NewClientArgs) (opamp.Client, error) {
 		return nil, fmt.Errorf("failed to create updaterManager: %w", err)
 	}
 
+	// Propagate TLS config to repoteManager agent
+	tlsCfg, err := args.Config.ToTLS()
```

Comment:
does this just return nil if there's no tls config? (Edit: it must because the unit tests work)

Reply:
Yes, nil is an acceptable TLS config so we just set it accordingly.